stdlib/flux_test.go

```diff
@@ -34,6 +34,7 @@ var skip = map[string]string{
 	"task_per_line":               "join produces inconsistent/racy results when table schemas do not match (https://github.com/influxdata/flux/issues/855)",
 	"rowfn_with_import":           "imported libraries are not visible in user-defined functions (https://github.com/influxdata/flux/issues/1000)",
 	"string_trim":                 "cannot reference a package function from within a row function",
+	"integral_columns":            "aggregates changed to operate on just a single columnm.",
```

Comment:
Should we update integral to operate on a single column as well?

Reply:
that's what I did.  this is a skip-test list.  I thought we could bring this back later if we deliver on a multi-aggregate function.